crates/core/src/mock/fixtures/responses/starknet_getTransactionByHash.json

```diff
@@ -719,7 +719,7 @@
       "0x076e91a117d68549b7c7be395f1bd01596372f2ac631bd6ce6202430654434e",
       "0x04ef32bc4fd31910b365bff935637cc2b4a084c73a9bbd91e6f5e4fd6062deb0"
     ],
-    "transaction_hash": "0x03204b4c0e379c3a5ccb80d08661d5a538e95e2960581c9faf7ebcf8ff5a7d3c",
+    "transaction_hash": "0x3204b4c0e379c3a5ccb80d08661d5a538e95e2960581c9faf7ebcf8ff5a7d3c",
```

Comment:
some issue here with leading zeroes, this isn't great, as I had to duplicate two routes solely because of leading zeroes, if you have a better way Id like to take it